bin/build-drivers/src/build_drivers/common.clj

```diff
@@ -33,14 +36,54 @@
 (defn driver-edn-filename [driver]
   (u/filename (driver-project-dir driver) "deps.edn"))
 
-(defn- ->absolute [driver path]
+(defn current-deps-project-root
+  "Returns the root of the current `deps.edn` project (ex: a third party driver, or Metabase itself is running from the
+  core project root).  See [[clojure.tools.deps.alpha/find-edn-maps]] for justification of this impl (we use the same
+  underlying var here)."
+  []
+  deps.dir/*the-dir*)
+
+(defn- ->absolute [_driver path]
   (if (u/absolute? path)
     path
-    (u/filename (driver-project-dir driver) path)))
+    (u/filename (current-deps-project-root) path)))
 
 (defn driver-edn [driver edition]
   (let [edn      (deps/merge-edns ((juxt :root-edn :project-edn) (deps/find-edn-maps (driver-edn-filename driver))))
         combined (deps/combine-aliases edn #{edition})]
     (-> (deps/tool edn combined)
         ;; make sure :paths are absolute
         (update :paths (partial mapv (partial ->absolute driver))))))
+
+(defn get-jar-entry ^ZipEntry [^String jar-path ^String filename]
+  (with-open [zip-file (ZipFile. jar-path)]
+    (first
+      (filter
+        (fn [^ZipEntry zip-entry]
+          (= (str zip-entry) filename))
+        (enumeration-seq (.entries zip-file))))))
```

Comment:
It would be a little more idiomatic here to just use `some` instead of `filter` + `first`